### PR TITLE
This fix #131 by adding a 'vide-wrapper' class to the video wrapper div

### LINKED ISSUE #131: container class
Nice plugin!

While using it, there is maybe one thing that would be a nice simple addition, to put a Vide-something css class on the container, for easy targetting.

Cheers,

Spoutnik

### PR DESCRIPTION
**Pull request by blckshrk**

### CODE REVIEW

File: src/jquery.vide.js [hunk 1]
```diff
@@ -232,22 +232,24 @@
...
-      '-webkit-background-size': 'cover',
-      '-moz-background-size': 'cover',
-      '-o-background-size': 'cover',
-      'background-size': 'cover',
-      'background-color': settings.bgColor,
-      'background-repeat': 'no-repeat',
-      'background-position': position.x + ' ' + position.y
-    });
+    $wrapper = vide.$wrapper = $('<div>')
+      .addClass('vide-wrapper')
```
Comment (VodkaBears) [inline src/jquery.vide.js]: Let's make it customizable. Define the `className` option with an empty string as a default value, please.

**Comment by blckshrk** [comment]
Hi,

I've implemented a className option with a default empty value as requested, but I was thinking that maybe it could more logical to have a class by default. I think this is the more asked case and if conflicts occurs or anything else, one could remove the class by setting the className option to an empty string.

You don't think so ?

**Comment by VodkaBears** [comment]
Thanks. That's enough.

**Comment by VodkaBears** [comment]
I will update.

**Pull request merged**